packages/appstore/src/components/modals/wallets-migration-failed/__tests__/wallets-migration-failed.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import WalletsMigrationFailed from '../wallets-migration-failed';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+}));
+
+const mockRootStore = mockStore({
+    traders_hub: {
+        is_wallet_migration_failed: false,
+        setWalletsMigrationFailedPopup: jest.fn(),
+    },
+});
```

Comment:
@sergei-deriv Please use the `mockStore` inside each test case 🙏🏻 Also you don't need to override the default values, They are already there.